src/components/application_manager/src/rpc_handler_impl.cc

```diff
@@ -277,7 +277,8 @@ bool RPCHandlerImpl::ConvertMessageToSO(
                     "Convertion result: "
                         << result << " function id "
                         << output[jhs::S_PARAMS][jhs::S_FUNCTION_ID].asInt());
-      if (!hmi_so_factory().attachSchema(output, false)) {
+      if (!hmi_so_factory().attachSchema(output,
+                                         /*remove_fake_parameters=*/true)) {
```

Comment:
@ZhdanovP Remove this ugly comment.

Reply:
@AByzhynar fixed f3ef0f9285e8dfa8e4ae1607038b9b73f5b4f1fc